crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -208,42 +166,9 @@ static int is_ec_fips_approved(int curve_nid) {
   }
 }
 
-// Updates the service indicator if the elliptic curve contained in |eckey|
-// is FIPS approved, otherwise does nothing.
-void EC_KEY_keygen_verify_service_indicator(EC_KEY *eckey) {
-  int curve_nid = EC_GROUP_get_curve_name(eckey->group);
-  if (is_ec_fips_approved(curve_nid)) {
-    FIPS_service_indicator_update_state();
-  }
-}
-
-void EVP_PKEY_keygen_verify_service_indicator(const EVP_PKEY *pkey) {
-  if (pkey->type == EVP_PKEY_RSA || pkey->type== EVP_PKEY_RSA_PSS) {
```

Comment:
Why was EVP_PKEY_RSA_PSS removed? I understand we support it: https://github.com/awslabs/aws-lc/pull/140 and ACVP-tested it.

Reply:
Thank you, Samuel. So I'm curious, how were the ACVP tests passing for PSS if we didn't have this check in the `if` condition.

Reply:
Oh we already had the check, I just forgot to add it back in when porting over BoringSSL's changes. 
But on a side note, I think this only checks if the RSA key is a PSS type in the service indicator, the ACVP functionality shouldn't be effected by it either way.

Reply:
We discussed offline. The ACVP tests were added in https://github.com/awslabs/aws-lc/pull/564/commits/56917817a32e5f0012a0f9e29fbdd7e403f940d5 when the check was added back, see https://github.com/awslabs/aws-lc/blob/56917817a32e5f0012a0f9e29fbdd7e403f940d5/crypto/fipsmodule/service_indicator/service_indicator_test.cc#L1196-L1225